src/AWS/Lambda/Events/ApiGateway/ProxyRequest.hs

```diff
@@ -0,0 +1,167 @@
+{-# LANGUAGE DuplicateRecordFields #-}
```

Comment:
FWIW I'm fine if we add these to our [`default-extensions`](https://github.com/Nike-Inc/hal/blob/553b2da12f0c1c059f74f4848c2210c53ec6b0cd/package.yaml#L20), they're pretty harmless on their own.

Reply:
Agreed, this is a good candidate for a default.